Gauge/Source/GC.hs

```diff
@@ -43,10 +43,14 @@ supported = unsafePerformIO (readIORef supportedVar)
 
 supportedVar :: IORef Bool
 supportedVar = unsafePerformIO $ do
+#if __GHCJS__
+    let b = False
+#else
```

Comment:
can probably save one wrapping by switching this #else to #elif

Reply:
I will fix that.